nikola/nikola.py

```diff
@@ -1694,9 +1694,7 @@ def _register_templated_shortcodes(self):
         """
         self.register_shortcode('template', self._template_shortcode_handler)
 
-        builtin_sc_dir = resource_filename(
-            'nikola',
-            os.path.join('data', 'shortcodes', utils.get_template_engine(self.THEMES)))
+        builtin_sc_dir = str(resources.path('nikola')) if sys.version_info.minor == 8 else str(resources.files('nikola').joinpath(os.path.join('data', 'shortcodes', utils.get_template_engine(self.THEMES))))
```

Comment:
The two things are not equivalent, this is why a function in utils would be a better choice.